evm/precompile/router/router.go

```diff
@@ -35,22 +37,36 @@ func (c *Contract) SendMessage(
 	messageID string,
 	namespace string,
 ) (bool, error) {
+	log.Logger.Debug().Msg("inside SendMessage precompile function called")
 	pCtx := vm.UnwrapPolarContext(ctx)
-	err := c.rtr.SendMessage(ctx, namespace, pCtx.MsgSender().String(), messageID, message)
-	if err != nil {
-		return false, err
+	if c.rtr != nil {
```

Comment:
nit: 
```
if c.rtr == nil {
    log.Logger.Debug()...
    return false, fmt.Error("nil router")
}
// happy path...
```

Reply:
will return to this eventually, but gonna leave as is for now